examples/azure-stack/kubernetes-azurestack.json

```diff
@@ -15,6 +15,9 @@
         "customCloudProfile": {
             "portalURL": ""
         },
+        "featureFlags": {
+            "enableTelemetry": true
```

Comment:
is this feature meant to be configurable by users? ie. users can opt out? The way I understood the intent of this change, this is to help the team track Azure stack aks-engine operations, not a feature for customers to get telemetry on their clusters. If that's the case, exposing it as an `exposeTelemetry` flag in the apimodel might be confusing for users (especially as a "featureFlag" since these are meant to be experimental knobs for feature development work). Thoughts?

Reply:
I think that introducing something like this via FeatureFlags is appropriate, and when it becomes more generally functional and defined (i.e., not just avail in Azure Stack) we can matriculate the feature elsewhere.

My 2¢

Reply:
I agree with Jack, this feature will at some point enable not only azurestrack to track deployments but also enable azure to track deployments. On azure however the user has to generate the guid used in the telemetry tracking.

Reply:
sounds good